docs/native_crash_collection.md

```diff
@@ -19,15 +28,39 @@ dependencies {
 }
 ```
 
-### Plugins
-
-1. The NDKCrashReporterPlugin handles the crash signals at the native level and reports them as errors by adding more explicit information (for example: backtrace, signal name, signal relevant error message) in the Datadog logs dashboard:
+2. Initialize the library with your application context, tracking consent, and the [Datadog client token][1] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Android RUM Collection][3] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][2] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][1]:
```

Comment:
There doesn't seem to be any other numbered steps. Maybe this is the same as the rum_collection.md page, which introduces line 20 with  "1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:"